File: mobile/lib/src/modules/auth/data/industries.dart
import 'package:flutter/foundation.dart';

@immutable
class Industry {
  const Industry({required this.id, required this.label, required this.emoji});

  final String id;
  final String label;
  final String emoji;
}

const List<Industry> kIndustries = [
  Industry(id: 'store', label: 'Store', emoji: '🛍️'),
  Industry(id: 'content-creators', label: 'Content Creators', emoji: '🎙️'),
  Industry(id: 'sport-fitness', label: 'Sport & Fitness', emoji: '🏋️'),
  Industry(id: 'creative', label: 'Creative', emoji: '🎨'),
  Industry(id: 'food-drink', label: 'Food & Drink', emoji: '🍕'),
  Industry(id: 'real-estate', label: 'Real Estate', emoji: '🏠'),
  Industry(id: 'beauty', label: 'Beauty', emoji: '💄'),
  Industry(id: 'events', label: 'Events', emoji: '🎤'),
  Industry(id: 'business', label: 'Business', emoji: '💼'),
  Industry(id: 'fashion', label: 'Fashion', emoji: '👗'),
  Industry(id: 'wellness-health', label: 'Wellness & Health', emoji: '🌿'),
  Industry(id: 'entertainment', label: 'Entertainment', emoji: '🎭'),
  Industry(id: 'education', label: 'Education', emoji: '📚'),
  Industry(id: 'music', label: 'Music', emoji: '🎵'),
  Industry(id: 'tech-gaming', label: 'Tech & Gaming', emoji: '🎮'),
  Industry(id: 'travel-activities', label: 'Travel & Activities', emoji: '🌴'),
  Industry(id: 'community', label: 'Community', emoji: '🤝'),
  Industry(id: 'other', label: 'Other', emoji: '✨'),
];

File: web/src/data/industries.ts
export interface Industry {
  id: string
  label: string
  emoji: string
}

export const INDUSTRIES: readonly Industry[] = [
  { id: 'store', label: 'Store', emoji: '🛍️' },
  { id: 'content-creators', label: 'Content Creators', emoji: '🎙️' },
  { id: 'sport-fitness', label: 'Sport & Fitness', emoji: '🏋️' },
  { id: 'creative', label: 'Creative', emoji: '🎨' },
  { id: 'food-drink', label: 'Food & Drink', emoji: '🍕' },
  { id: 'real-estate', label: 'Real Estate', emoji: '🏠' },
  { id: 'beauty', label: 'Beauty', emoji: '💄' },
  { id: 'events', label: 'Events', emoji: '🎤' },
  { id: 'business', label: 'Business', emoji: '💼' },
  { id: 'fashion', label: 'Fashion', emoji: '👗' },
  { id: 'wellness-health', label: 'Wellness & Health', emoji: '🌿' },
  { id: 'entertainment', label: 'Entertainment', emoji: '🎭' },
  { id: 'education', label: 'Education', emoji: '📚' },
  { id: 'music', label: 'Music', emoji: '🎵' },
  { id: 'tech-gaming', label: 'Tech & Gaming', emoji: '🎮' },
  { id: 'travel-activities', label: 'Travel & Activities', emoji: '🌴' },
  { id: 'community', label: 'Community', emoji: '🤝' },
  { id: 'other', label: 'Other', emoji: '✨' },
] as const
